tests/e2e/monovertex.go

```diff
@@ -88,13 +92,14 @@ func verifyMonoVertexReady(namespace, monoVertexName string) {
 
 }
 
-func verifyMonoVertexStatus(namespace, monoVertexName string, f func(numaflowv1.MonoVertexSpec, kubernetes.GenericStatus) bool) {
+func verifyMonoVertexStatus(namespace, monoVertexRolloutName string, f func(numaflowv1.MonoVertexSpec, kubernetes.GenericStatus) bool) string {
```

Comment:
is this name that it's returning being used anywhere? If not, maybe we can take it out?

Reply:
oh, I see you're using it above